examples/nest/gap_junction/inhibitory_network.py

```diff
@@ -64,23 +65,43 @@
 
 nest.ResetKernel()
 
-nest.SetKernelStatus({'resolution': 0.05, 'total_num_virtual_procs': threads})
-nest.SetKernelStatus({'max_num_prelim_iterations': 15, 'prelim_interpolation_order': 3, 'prelim_tol': 0.0001})
+nest.SetKernelStatus({'resolution': 0.05, 
+                      'total_num_virtual_procs': threads,
+                      'print_time': True})
 
-neuron = nest.Create('hh_psc_alpha_gap',n_neuron)
+# Settings for waveform relaxation
+# 'use_wfr': False uses communication in every step 
+# instead of an iterative solution
+nest.SetKernelStatus({'use_wfr': True,
+                      'wfr_comm_interval': 1.0,
+                      'wfr_tol': 0.0001,
+                      'wfr_max_iterations': 15,
+                      'wfr_interpolation_order': 3})
```

Comment:
Why do you have two subsequent calls to `SetKernelStatus`? Can't you call it once like this: 

    nest.SetKernelStatus({'resolution': 0.05, 
                          'total_num_virtual_procs': threads,
                          'print_time': True,
    # comments ...
                          'use_wfr': True,
                          'wfr_comm_interval': 1.0,
                          'wfr_tol': 0.0001,
                          'wfr_max_iterations': 15,
                          'wfr_interpolation_order': 3})

(Similar in the other example.)